### ISSUE TITLE
Decreased serum ficolin-3

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10269]
**Preferred term label:**
Decreased serum ficolin-3

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C4313036

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ficolin 3 deficiency

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000189
- label: Decreased circulating ficolin 3 concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: PMID:25662573
- def: Concentration of ficolin 3 in the blood circulation below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10269
- synonyms:
